src/test/mina_automation/executor.mli

```diff
@@ -0,0 +1,65 @@
+(* executor.mli *)
+open Async
+
+(** The type representing an executor *)
+
+
+module type AppPaths = sig
+  (** The path to app from repo root *)
+  val dune_name : string
+ 
+  (** The name of an app after debian/docker installation*)
+  val official_name : string
+end
+
+(** Module DockerContext provides an abstract type [t] representing the context for Docker operations. *)
+module DockerContext : sig
```

Comment:
I suggest we reorder the symbols in the interface so it matches the order we have in `executor.ml`. For example, `DockerContext` is defined before `AppPaths`.